app/uk/gov/hmrc/openidconnect/userinfo/connectors/AuthConnector.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2016 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.openidconnect.userinfo.connectors
+
+import play.api.Logger
+import uk.gov.hmrc.openidconnect.userinfo.config.WSHttp
+import uk.gov.hmrc.play.auth.microservice.connectors.ConfidenceLevel._
+import uk.gov.hmrc.play.config.ServicesConfig
+import uk.gov.hmrc.play.http.{HeaderCarrier, HttpGet}
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+trait AuthConnector extends uk.gov.hmrc.play.auth.microservice.connectors.AuthConnector {
+  val authBaseUrl: String
+  val http: HttpGet
```

Comment:
:) As per the chat - this is a pattern we've got .. really everywhere in our codebase.  I think it's worth having a larger discussion around it if people feel like it's going to improve code quality/readability significantly. 

But while we're doing that long-term discussion, I'm going to merge this PR (so edgar isn't blocked).